test/units/galaxy/test_collection.py

```diff
@@ -198,6 +199,48 @@ def manifest(manifest_info):
             yield fake_file, sha256(b_data).hexdigest()
 
 
+@pytest.fixture()
+def server_config(monkeypatch):
+    monkeypatch.setattr(C, 'GALAXY_SERVER_LIST', ['server1', 'server2', 'server3'])
+
+    default_options = dict((k, None) for k, v in SERVER_DEF)
+
+    server1 = dict(default_options)
+    server1.update({'url': 'https://galaxy.ansible.com/api/', 'validate_certs': False})
+
+    server2 = dict(default_options)
+    server2.update({'url': 'https://galaxy.ansible.com/api/', 'validate_certs': True})
+
+    server3 = dict(default_options)
+    server3.update({'url': 'https://galaxy.ansible.com/api/'})
```

Comment:
You and the same URL into all the objects. Maybe add it to the base one only?

Reply:
Fixed